### ISSUE TITLE
Invalid Node failure of TensorRT 8.6 when running a ONNX model including CumSum Layer on GPU MX330

### ISSUE THREAD

**Issue by hongliyu0716** [NVIDIA/TensorRT#3425]
## Description

I tried to run an ONNX model, but it fails with the error below.
```
Traceback (most recent call last):
  File "cumsum_crash.py", line 18, in <module>
    raise RuntimeError(error_msg)
RuntimeError: In node 0 (parseGraph): INVALID_NODE: Invalid Node - node_of_y
Axis input for CumSum must be an initializer! Try applying constant folding on the model using Polygraphy: https://github.com/NVIDIA/TensorRT/tree/master/tools/Polygraphy/examples/cli/surgeon/02_folding_constants
```

## Environment

**TensorRT Version**: 8.6.1.6

**NVIDIA GPU**: NVIDIA GeForce MX330

**NVIDIA Driver Version**: 470.182.03

**CUDA Version**: 11.4

**CUDNN Version**: 8.9.5

Operating System: Ubuntu 18.04

Python Version: 3.8

## Relevant Files

**Model link**: https://github.com/hongliyu0716/onnx_model/blob/main/cumsum_crash.onnx

## Steps To Reproduce

1. Download the model
2. Execute the script
```python
import onnx
import tensorrt as trt
import pycuda.driver as cuda
import pycuda.autoinit
import numpy as np

onnx_path = 'cumsum_crash.onnx'
model = onnx.load(onnx_path)
onnx.checker.check_model(model)

builder = trt.Builder(trt.Logger(trt.Logger.WARNING))
network = builder.create_network(1 << int(trt.NetworkDefinitionCreationFlag.EXPLICIT_BATCH))
config = builder.create_builder_config()
config.set_memory_pool_limit(trt.MemoryPoolType.WORKSPACE, 2 << 30)
parser = trt.OnnxParser(network, trt.Logger(trt.Logger.WARNING))
    
with open(onnx_path, 'rb') as f:
    if not parser.parse(f.read()):
        error_msg = ""
        for error in range(parser.num_errors):
            error_msg += str(parser.get_error(error))
        raise RuntimeError(error_msg)
```

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**: yes

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`): yes
![cumsum_polygraphy](https://github.com/NVIDIA/TensorRT/assets/149884162/f448a88e-c72e-4e2f-97a0-7f5c3091b91f)

**Comment by zerollzeng** [comment]
> ```
> Axis input for CumSum must be an initializer! Try applying constant folding on the model using Polygraphy: https://github.com/NVIDIA/TensorRT/tree/master/tools/Polygraphy/examples/cli/surgeon/02_folding_constants
> ```

Could you please try constant folding the model as the error suggests?

**Comment by hongliyu0716** [comment]
Sure, I have already attempted constant folding the model as the error suggests, but unfortunately, I am still facing the same error.

**Comment by zerollzeng** [comment]
I think you need to modify the source code then re-export the onnx.

**Comment by Thrsu** [comment]
> I think you need to modify the source code then re-export the onnx.

Thank you for your response. I understand your suggestion to modify the source code and re-export the ONNX model. But I have tested the ONNX model with different frameworks and it has produced the expected results consistently.  And I have examined the [official documentation of ONNX](https://onnx.ai/onnx/operators/onnx__CumSum.html#inputs) based on the error message, I found no mention of the limitation 'Axis input for CumSum must be an initializer.' According to the documentation, the `axis` should be of type Tensor, not an initializer.

**Comment by hongliyu0716** [comment]
> I think you need to modify the source code then re-export the onnx.

Thank you for your response. I understand your suggestion to modify the source code and re-export the ONNX model. But I have tested the ONNX model with different frameworks and it has produced the expected results consistently. And I have examined the [official documentation of ONNX](https://onnx.ai/onnx/operators/onnx__CumSum.html#inputs) based on the error message, I found no mention of the limitation 'Axis input for CumSum must be an initializer.' According to the documentation, the axis should be of type Tensor, not an initializer.
